packages/js/product-editor/src/blocks/taxonomy/use-taxonomy-search.ts

```diff
@@ -69,6 +69,7 @@ const useTaxonomySearch = (
 					taxonomyName
 				);
 			}
+			setIsSearching( false );
 		} catch ( e ) {
 			setIsSearching( false );
 		}
```

Comment:
This works. One possible alternative would be to put it in a `finally` block and change the `catch` to show an alert that an error occurred...

```ts
} finally {
    setIsSearching( false );
}
```

But, I'm okay with keeping things exactly as you have them for now.

Reply:
Good call! I changed that in the commit [906bdbc](https://github.com/woocommerce/woocommerce/pull/40073/commits/906bdbcf90d3ad098942f11fcc190a160c1cfcbf)